backend/import-service/lib/import-service-stack.ts

```diff
@@ -28,6 +29,15 @@ export class ImportServiceStack extends cdk.Stack {
       autoDeleteObjects: true, // For development only
     });
 
+    // Get reference to existing SQS queue
+    const queueArn = `arn:aws:sqs:${this.region}:${this.account}:catalogItemsQueue`;
+    const queueUrl = `https://sqs.${this.region}.amazonaws.com/${this.account}/catalogItemsQueue`;
+
+    const catalogItemsQueue = sqs.Queue.fromQueueAttributes(this, 'ImportCatalogItemsQueue', {
+      queueUrl,
```

Comment:
redundant. You can use `Queue.fromQueueArn` and get `catalogItemsQueue.queueUrl` on instance without predefining it.